Client/Frontend/Widgets/TwoLineCell.swift

```diff
@@ -4,11 +4,14 @@
 
 import UIKit
 
-private let ImageSize: CGFloat = 24
-private let ImageMargin: CGFloat = 12
-private let TextColor = UIAccessibilityDarkerSystemColorsEnabled() ? UIColor.blackColor() : UIColor(rgb: 0x333333)
-private let DetailTextColor = UIAccessibilityDarkerSystemColorsEnabled() ? UIColor.darkGrayColor() : UIColor.grayColor()
-private let DetailTextTopMargin = CGFloat(5)
+struct TwoLineCellUX {
+    static let ImageSize: CGFloat = 25
+    static let BorderViewMargin: CGFloat = 13
+    static let BorderWidth: CGFloat = 32
```

Comment:
`BorderWidth` sounds like, well, the width of the border, so I'd rename this. Maybe `BorderFrameSize`?